.gitignore

```diff
@@ -75,19 +78,30 @@ target/
 # celery beat schedule file
 celerybeat-schedule
 
-# dotenv
-.env
+# SageMath parsed files
+*.sage.py
 
-# virtualenv
+# Environments
+.env
+.venv
+env/
 venv/
 ENV/
-env2/
+env.bak/
+venv.bak/
 
 # Spyder project settings
 .spyderproject
+.spyproject
 
 # Rope project settings
 .ropeproject
 
-*.swp
-*.swo
```

Comment:
Do you want to get rid of these? They are vi's swap files

Reply:
Good catch. I think we want to keep swap files out of version control.
I've replaced it in the latest commit.